Test.groovy

```diff
@@ -20,8 +30,11 @@ pipeline {
   }
   post {
     always {
-      sh "docker-compose down"
-      junit "**/test-report.xml"
+      script {
+        sh "docker-compose down"
+        sh "rm -Rf fixtures"
```

Comment:
We clean up the fixtures in the end of the pipeline because otherwise the folder would keep growing with new fixtures migrations, all of which would be the same migrations but with different timestamps.